ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1693,13 +1722,23 @@ def test_ipahealthcheck_certmongerca(self, getcert_ca):
             self.master, "ipahealthcheck.ipa.certs", "IPACertmongerCA",
         )
         assert returncode == 1
+        version = tasks.get_healthcheck_version(self.master)
+        returncode, data = run_healthcheck(
+            self.master, "ipahealthcheck.ipa.certs", "IPACertmongerCA",
+        )
+        assert returncode == 1
```

Comment:
The lines 1721-1724 and 1726-1729 are a copy-paste, probably unwanted.

Reply:
Removed the unwanted code here.